components/clickup/actions/create-task/create-task.js

```diff
@@ -0,0 +1,167 @@
+const common = require("../common.js");
+
+module.exports = {
+  ...common,
+  key: "clickup-create-task",
+  name: "Create Task",
+  description: "Creates a new task",
+  version: "0.0.1",
+  type: "action",
+  props: {
+    ...common.props,
+    list: {
+      propDefinition: [
+        common.props.clickup,
+        "list",
+        (c) => ({
+          folder: c.folder,
+          space: c.space,
+        }),
+      ],
+    },
+    name: {
+      type: "string",
+      label: "Name",
+      description: "New task name",
+    },
+    description: {
+      type: "string",
+      label: "Description",
+      description: "New task description",
+      optional: true,
+    },
+    assignees: {
+      propDefinition: [
+        common.props.clickup,
+        "assignees",
+        (c) => ({
+          workspace: c.workspace,
+        }),
+      ],
+    },
+    tags: {
+      propDefinition: [
+        common.props.clickup,
+        "tags",
+        (c) => ({
+          space: c.space,
+        }),
+      ],
+    },
+    status: {
+      propDefinition: [
+        common.props.clickup,
+        "status",
+        (c) => ({
+          list: c.list,
+        }),
+      ],
+    },
+    dueDate: {
+      type: "string",
+      label: "Due Date",
+      description:
+        "The date by which you must complete the task. Use UTC time in milliseconds (ex. 1508369194377)",
+      optional: true,
+    },
+    dueDateTime: {
+      type: "boolean",
+      label: "Due Date Time",
+      description:
+        "Set to true if you want to enable the due date time for the task",
+      optional: true,
+    },
+    timeEstimate: {
+      type: "string",
+      label: "Time Estimate",
+      description: "Use milliseconds",
+      optional: true,
+    },
+    startDate: {
+      type: "string",
+      label: "Start Date",
+      description:
+        "The start date of the task. Use UTC time in milliseconds (ex. 1567780450202)",
+      optional: true,
+    },
+    startDateTime: {
+      type: "boolean",
+      label: "Start Date Time",
+      description: "Select true if you want to enable the start date time",
+      optional: true,
+    },
+    notifyAll: {
+      type: "boolean",
+      label: "Notify All",
+      description:
+        "If notify_all is true, creation notifications will be sent to everyone including the creator of the task.",
```

Comment:
In the description, you should refer to this by label, not by name. The user can't see the name.

Reply:
Updated